### PR TITLE
fix(dom-api): adjust $addClass for IE11 svgs

### LINKED ISSUE #1079: classList does not exist on SVG elements in IE11
**Stencil version:**

```
 @stencil/core@0.13.0-0
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**

SVGs that have classes applied to them via stencil throw the error:
`Unable to get property 'add' of undefined or null reference ` in IE11 because it appears [IE11 does not support `classList` on SVG elements](https://caniuse.com/#feat=classlist). That unfortunately breaks [domAPI.$addClass](https://github.com/ionic-team/stencil/blob/62292d37d3072476c27179b83c10c8d63dfc97c2/src/renderer/dom-api.ts#L59) but only for SVGs....in IE11 (/shakeFist).

**Expected behavior:**

Class is applied or IE11 burns in effigy ....whichever comes first.

**Steps to reproduce:**

If needed I can probably cobble together some code though it seems pretty straight-forward. At least they are pseudo-documented edge cases relegated to SVG or MathML elements in IE11. 

**Related code:**
```tsx
// insert any relevant code here
```

**Other information:**

Please let me know if you need more information.

### PR DESCRIPTION
**Pull request by cary-smith**
[IE11 does not support classList on SVG](https://caniuse.com/#search=classList) (or MathML)

fixes #1079

### CODE REVIEW

File: src/renderer/dom-api.ts [hunk 1]
```diff
@@ -55,8 +55,19 @@ export function createDomApi(App: AppGlobal, win: any, doc: Document): DomApi {
     $appendChild: (parentNode: Node, childNode: Node) =>
       parentNode.appendChild(childNode),
 
-    $addClass: (elm: Element, cssClass: string) =>
-      elm.classList.add(cssClass),
+    $addClass: (elm: Element, cssClass: string) => {
+      if (!!elm.classList) {
+        elm.classList.add(cssClass);
+      } else if (elm.nodeName === "SVG") {
```
[the remark this answers is not in the document]
Comment (adamdbradley) [reply src/renderer/dom-api.ts]: Yup, add the `__BUILD_CONDITIONALS__` and I think it's good. The build conditional will make sure that code is only added for the es5 builds, and if your app has svgs.

**Comment by manucorporat** [comment]
It might be a good idea to add a karma test that reproduces the issue, @cary-smith i can help u with that. Could you share a simple component that reproduces the issue?

**Comment by cary-smith** [comment]
@manucorporat I will indeed. May take me a minute since I have a few things in the air at the moment.

**Comment by manucorporat** [comment]
@cary-smith hi! i noticed now the PR includes lots of stylistic changes, could you revert those? it would be great to only have changes in the `addClass` function! thanks!

**Comment by cary-smith** [comment]
@manucorporat Found the time. Test is in.

**Comment by cary-smith** [comment]
@manucorporat - uh oh.....Let me go fix that! I think I had `prettier` turned on

**Comment by cary-smith** [comment]
Cleaned up. Please let me know if I have missed anything and sorry for the reformat.

**Comment by adamdbradley** [comment]
merged! thanks!

**Comment by cary-smith** [comment]
Thank you!

**Pull request merged**